### ISSUE TITLE
A11y : Cannot select Log lines via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46485]
**Steps to reproduce**:

Attempt to navigate the labels / values inside the Log Browser using keyboard-only navigation

**Actual Result**:

Log browser not usable - workaround would be to take note of labels and value and then use the text editor input

**Expected Result**

Better accessibility for keyboard-only navigators

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by matyax** [comment]
@zuchka What do you mean with "Log browser" in this issue? I would assume it's the list of log lines, but the "labels / values" throw me off. Thanks!

**Comment by zuchka** [comment]
@matyax apologies for the delay. I do believe this was in reference to the list of log lines. @yaelleC might remember the specifics. I created all these issues from a doc, but I did miss this particular bug-hunting session...

**Comment by matyax** [comment]
Thanks! I assumed this issue was about this:

![imagen](https://user-images.githubusercontent.com/1069378/204588279-12792139-8a80-47ba-a3ed-c963a114d8cb.png)

But those are technically log lines (AFAIK), not labels and values.

**Comment by zuchka** [comment]
@matyax I think you are correct here. That looks to be the major blocker with keyboard navigation afaik. Also, keyboard navigation looks pretty solid once inside the expanded log line:

![CleanShot 2022-11-30 at 14 34 00](https://user-images.githubusercontent.com/37156449/204937640-c10ab131-c48d-4c46-b708-abc38781ec35.gif)

**Comment by matyax** [comment]
Thank you! Hopefully we will be working on this soon.
